packages/viewer/src/components/viewer-pin-label/vertex-pin-label.css

```diff
@@ -0,0 +1,34 @@
+:host {
+  /**
+   * @prop --viewer-annotations-pin-label-background-color: A CSS color that
+   * specifies the color of the label background
+   */
+   --viewer-annotations-pin-label-background-color: var(--blue-200);
+
+  /**
+   * @prop --viewer-annotations-pin-label-background-color: A CSS color that
+   * specifies the color of the label background
+   */
+   --viewer-annotations-pin-label-border-color: var(--blue-600);
+
+  overflow: hidden;
+  pointer-events: none;
+}
+:host([mode='']) .pin-label {
+  color: var(--white);
+  background: var(--neutral-900);
+}
+
+
+.pin-label {
+  position: absolute;
+  background: var(--viewer-annotations-pin-label-background-color);
+  border: 2px solid var(--viewer-annotations-pin-label-border-color);
+  border-radius: 0.25rem;
+  padding: 0.375rem 0.5rem;
```

Comment:
Similarly it might be nice to customize some of these properties to remove the border/border radius or reduce padding on the label